dashboard/config/locales/en.yml

```diff
@@ -692,82 +692,156 @@ en:
+    donate:
+      image_alt: 'A child pointing at a laptop screen showing a Code.org puzzle.'
+    elementary:
+      image_alt: 'A bumble bee and squirrel standing in a field in front of a rainbow.'
+    express:
+      image_alt: 'Plants vs Zombies zombie and plant standing next to Flappy Bird.'
     flappy:
       title: 'Flappy Code'
       body: 'Wanna write your own game in less than 10 minutes? Try our Flappy Code tutorial!'
-    20-hour:
-      title: 'Accelerated Course'
-      body: 'This 20-hour course covers the core computer science and programming concepts in courses 2-4. The course is designed for use with ages 10-18. Check out courses 2-4 for a more complete experience!'
-      body_short: 'Learn basic computer science in an accelerated version of courses 2-4.'
-    hoc2014:
-      try_new_tutorial: 'Try our new Hour of Code tutorial in beta'
-      try_frozen: 'Artist with Anna and Elsa, from Frozen'
+      image_alt: 'A yellow bird flying between green pipes.'
+    frozen:
+      title: "Frozen"
+      image_alt: 'Anna and Elsa from the movie Frozen. Between them is an image of Anna drawing a pretty shape in the ice using her ice skates.'
     gallery:
       title: "Gallery"
       body: "View a gallery of app and artist images created by our users"
+      image_alt: 'An artist drawing a lot of circles and a cat, bat, and ghost standing in a winter holiday land.'
     gamelab:
       title: "Game Lab"
       body: "Game Lab is a programming environment where you can make simple animations and games with objects and characters that interact with each other."
       audience: "Ages 13+"
       cta: "Learn how"
+      image_alt: 'Many cartoon people and animals standing on a sidewalk and smiling.'
+    hello-world-food:
+      image_alt: 'Many animals, aliens and emojis orbiting the Earth.'
+    hello-world:
+      title: 'Hello World'
+      body: 'Choose from four fun themes to code interactive characters in a world you create.'
+    hero:
+      title: "Minecraft"
+      body: "Minecraft is back for the Hour of Code with a brand new activity! Journey through Minecraft with code."
+      image_alt: 'An Iron Golem from Minecraft standing in front of an overhead view of Anna solving a puzzle.'
     hoc-encryption:
       title: "Simple Encryption"
       body: "Students will learn techniques to create and crack encrypted messages."
-    weblab:
-      title: "Web Lab (beta)"
-      body: "Web Lab is a programming environment where you can make simple web pages using HTML and CSS. Design your web pages and share your site in seconds."
-      audience: "Ages 13+"
-      cta: "Learn how"
-    frozen:
-      title: "Frozen"
-    starwars:
-      title: "Star Wars"
+      image_alt: 'A bar graph of letter frequencies in the English language.'
+    hoc:
+      title: 'Classic Maze'
+      body: 'Try the basics of computer science. Millions have given it a shot.'
+      image_alt: 'A collage of code, Mark Zuckerberg, and Angry Birds.'
```

Comment:
nit: Should we try to make them all sentences? Eg. `A collage shows a code snippet, Mark Zuckerberg, and Angry Birds."